### PR TITLE
Feature/view technology

### PR DESCRIPTION
**Pull request by tsr-thulio**

### CODE REVIEW

File: src/main/webapp/js/controllers.js [hunk 1]
```diff
@@ -48,28 +48,34 @@ angular
                     };
 
                     function mockList() {
-                        var descr = "Mussum ipsum cacilds, vidis litro abertis. Consetis adipiscings elitis. Pra lá , depois divoltis porris, paradis. Paisis, filhis, espiritis santis.";
+                        var descr = "Mussum ipsum cacilds, vidis litro abertis. Consetis adipiscings elitis. Pra lÃ¡ , depois divoltis porris, paradis. Paisis, filhis, espiritis santis.";
```
Comment (larruda) [inline src/main/webapp/js/controllers.js]: Is the file encoding correct?

File: src/main/webapp/js/controllers.js [hunk 2]
```diff
@@ -93,8 +98,41 @@ angular
...
+                        var idTechnology = getParameterByName('id');
+                        var req = {
+                            id : idTechnology
+                        };
+                        gapi.client.rest.getTechnology(req).execute(
+                                function(data) {
+                                    $scope.name = data.name;
+                                    $scope.description = data.description;
+                                    $scope.citDesc = data.citDesc;
+                                    $scope.image = data.image;
```
Comment (larruda) [inline src/main/webapp/js/controllers.js]: Please avoid mentions to CI&T on the source-code. This application should be company agnostic as much as possible.

File: src/main/webapp/viewTech.html [hunk 3]
```diff
@@ -22,25 +22,16 @@
...
 				<div id="div_image" class="col-xs-6">
-					<img src="img/teste.png" class="img-responsive">
+					<img src={{image}} class="img-responsive">
 				</div>
 				<div id="div_teste" class="container">
 					<div id="div_image_name" class="col-xs-6">
-						<h1>LOREM</h1>
-						<h4>CI&T Description</h4>
+						<h1>{{name}}</h1>
+						<h4>{{citDesc}}</h4>
```
Comment (larruda) [inline src/main/webapp/viewTech.html]: Same as above.

File: src/main/webapp/js/controllers.js (hunk 1 above)
Comment (tsr-thulio) [inline src/main/webapp/js/controllers.js]: My enconding configurations are ok, with "Text File: UTF-8" and "New Line: Unix". Anyway  i fixed the line and commited again.

File: src/main/webapp/js/controllers.js (hunk 2 above)
Comment (tsr-thulio) [inline src/main/webapp/js/controllers.js]: done

File: src/main/webapp/viewTech.html (hunk 3 above)
Comment (tsr-thulio) [inline src/main/webapp/viewTech.html]: done

**Pull request merged**